tests/ui/async-await/in-trait/nested-rpit.rs

```diff
@@ -1,7 +1,10 @@
 // edition: 2021
-// known-bug: #105197
-// failure-status:101
-// dont-check-compiler-stderr
+// [current] known-bug: #105197
+// [current] failure-status:101
+// [current] dont-check-compiler-stderr
+// [next] compile-flags: -Zlower-impl-trait-in-trait-to-assoc-ty
```

Comment:
I am skeptical that this PR passes... I'll look into it though.

Reply:
Sorry, skeptical **why** this PR passes.

Reply:
Oh... this fails because we have no `predicates_of` for RPITITs. That doesn't seem right :thinking: 

Let's leave this as not updated currently?